clayui.com/package.json

```diff
@@ -55,6 +56,7 @@
 		"gatsby-plugin-react-helmet": "^3.3.10",
 		"gatsby-plugin-sass": "^2.3.12",
 		"gatsby-plugin-typescript": "^2.4.18",
+		"gatsby-react-router-scroll": "3.0.0",
```

Comment:
@pat270 from what I understand it needs is like `resolutions` too so you can force the use of this version in Gatsby or is that just enough?

Reply:
This worked for me, but I can add the resolutions line.

Reply:
No problem, I say this because I see two versions of the package in yarn.lock:

https://github.com/liferay/clay/pull/3676/files?file-filters%5B%5D=.js&file-filters%5B%5D=.json&file-filters%5B%5D=.md&file-filters%5B%5D=.scss#diff-8ee2343978836a779dc9f8d6b794c3b2R11141